haskell/test/Lorentz/Contracts/Nettest/StablecoinClientTest.hs

```diff
@@ -38,9 +37,9 @@ test_stablecoinClientScenario =
 -- | Check that all the `stablecoin-client` commands work.
 stablecoinClientScenario :: StablecoinT m ()
 stablecoinClientScenario = do
-  Moneybag originator <- asks $ ccMoneybag . SC.scCleveland
-
   comment "Creating roles"
+  originator <- newAddress "originator"
+  transfer originator [tz|5|]
```

Comment:
```haskell
originator <- refillable $ newAddress "originator"
```
?

The downside refillables are somewhat slower. The upside is they don't require hard-coded amounts.

Reply:
So, I just realized this doesn't work (see https://github.com/tqtezos/stablecoin/pull/198#discussion_r998515700), and it's because the `deploy` command below doesn't originate the contract via Cleveland.

It [actually calls the `stablecoin-client deploy`](https://github.com/tqtezos/stablecoin/blob/bc167c9d043dbcb0d95c297e0b73fde7cf314ee4/haskell/src/Stablecoin/Client/Cleveland/StablecoinImpl.hs#L99-L112) executable, which in turn will originate the contract via morley-client.

So the "refillable" mechanism is bypassed completely.

So our only options are to (1) transfer 5 tz up front, or (2) use cleveland's internals and use the moneybag account directly. Both have their pros and cons... I'm honestly ok with either one.

Reply:
Ah, I see. Ran into a similar problem with tzbtc smoke tests, with the same resolution. As for using the moneybag vs hard-coded amount, the former is more robust wrt changes in fees. I don't really have a decisive argument for one or the other, either.

Reply:
> the former is more robust wrt changes in fees

Yup, agreed, I ended up doing just that.